### PR TITLE
feat(DDI-1151): Added example for formatting options

### PR DESCRIPTION
**Pull request by Radha-Alberta**
Added example in checkbox component to bold the checkbox label

### CODE REVIEW

File: libs/docs/src/components/common/Checkbox.stories.mdx [hunk 1]
```diff
@@ -327,6 +327,56 @@ errorCheckboxTemplate.args = {
...
+    <CodeSnippet
+      code={`
+        <goa-checkbox
+          goaValue
+          name="desserts"
+          value="10"
+          [formControl]="reactiveFormCtrl"
+          [value]="reactiveFormCtrl.value"
+        >
+        <b>Ice Cream</b>
```
Comment (ArakTaiRoth) [inline libs/docs/src/components/common/Checkbox.stories.mdx:357]: Just indent this, please.

**Comment by chrisolsen** [comment]
You will need to add a space in your commit message between the colon and `A` for this change to be properly recognized.

File: libs/docs/src/components/common/Checkbox.stories.mdx (hunk 1 above)
Comment (Radha-Alberta) [reply libs/docs/src/components/common/Checkbox.stories.mdx]: Done

**Review by ArakTaiRoth — APPROVED** [review]

**Pull request merged**